src/lib/holocene/input/chip-input.svelte

```diff
@@ -62,12 +63,16 @@
   const handlePaste = (e: ClipboardEvent) => {
     e.preventDefault();
     const clipboardContents = e.clipboardData.getData('text/plain');
-    values.update((previous) => [...previous, ...clipboardContents.split(',')]);
+    values.update((previous) => [
+      ...previous,
+      ...clipboardContents.split(',').map((content) => content.trim()),
```

Comment:
Maybe we `.trim()` clipboardContents` too?

Reply:
I think we should be covered with just `content.trim()` as any leading/trailing whitespaces will just be added to the first and last "content".

Reply:
Oh yeah, very true.